secure_message_v2/views/batch_requests.py

```diff
@@ -27,3 +30,10 @@ def verify_password(username: str, password: str) -> Optional[str]:
 def mark_thread_for_deletion() -> tuple[str, int]:
     marked_for_deletion_by_closed_at_date()
     return "", 204
+
+
+@batch_request_bp.route("/threads", methods=["DELETE"])
+@auth.login_required
+def delete_threads() -> tuple[str, int]:
+    delete_threads_marked_for_deletion()
+    return "", 204
```

Comment:
I did initially expect to see a slightly different response for this requests for when we see deletions versus no deletions, but I'm not so sure thinking about it. A straight 204 might be ok for any batch requests regardless of if anything was actioned or not! I suppose if all depends if any calling service needs to make use of additional info. Which in the case of a cron job it doesn't.

Reply:
Because it was going back to a cron job and isn't needed or displayed anywhere a no content 204 felt fine for both. Adding logic and tests for no added value seems unnecessary, plus I would have to pass the count up the layers which I kind of didn't want to do

Reply:
Agreed. Feels appropriate given the use case.